### PR TITLE
Full App

### PR DESCRIPTION
**Pull request by mutinhiri**
In this Pr we added changes for the full app to work properly

### CODE REVIEW

File: src/Home.js [hunk 1]
```diff
@@ -0,0 +1,36 @@
...
+      <Header />
+      <div className="home-content">
+        <Routes>
+          <Route path="/" element={<Main />} exact />
+          <Route path="/appointment" element={<AppointmentForm />} />
+          <Route path="/appointments" element={<Appointment />} />
+          <Route path="/delete" element={<DeleteItem />} />
+          <Route path="/Detail/:id" element={<DetailPage />} />
+          <Route path="/AddItem" element={<AddItem />} />
+          {/* <Route path="/appointment-form" element={<AppointmentForm />} /> */}
```
Comment (MuriloRoque) [inline src/Home.js]: - [ ] Remove this commented code if you won't use it.

**Review by MuriloRoque — CHANGES_REQUESTED** [review]
Hi @mutinhiri, @Emmy-github-webdev,

Good job so far!
There are some issues that you still need to work on to prepare your project for the final evaluation but you are almost there!

## Suggested changes
_Check the comments under the review._

You can use as many of my suggestions as you want. If there is anything you would like to skip - feel free to do that. However, I strongly recommend you to take them into account as they can make your code better.

You can also consider:

- [ ] Add tests for your app, it's very important.

## Highlights 

- Very nice and clean code.
- Professional repository.

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

File: src/api/BaseURL.js [hunk 2]
```diff
@@ -0,0 +1,6 @@
+// const Development = 'http://127.0.0.1:8080/api/v1/';
```
Comment (christianotieno) [inline src/api/BaseURL.js:1]: Do we need this here? 🤔

File: src/components/AddItem/AddItem.js [hunk 3]
```diff
@@ -0,0 +1,57 @@
+/* eslint-disable react/jsx-props-no-spreading */
```
Comment (christianotieno) [inline src/components/AddItem/AddItem.js:1]: Why are we disbaling eslint for props spreading? 🤔

File: src/components/appointments/Appointments.js [hunk 4]
```diff
@@ -0,0 +1,52 @@
+/* eslint-disable react/no-array-index-key */
```
Comment (christianotieno) [inline src/components/appointments/Appointments.js:1]: Same case here for array index. I am sure there is a workaround. What do you think?

**Review by christianotieno — CHANGES_REQUESTED** [review]
Hi @mutinhiri & @Emmy-github-webdev, ✋🏾  

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!


 ## Required Changes ♻️
_Check the comments under the review._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.


------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have only 3 code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Review by MuriloRoque — APPROVED** [review]
Hi @mutinhiri, @Emmy-github-webdev,

In my opinion, your project is ready for the presentation!  There is nothing else to say other than... it's time to merge it :shipit: 

Good luck! 🎉 

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

**Pull request merged**